src/centos/stream8/amd64/Dockerfile

```diff
@@ -57,8 +57,14 @@ RUN dnf install --setopt tsflags=nodocs --refresh -y \
         wget \
         which \
         xz \
-        zlib-devel \
-    && \
+        zlib-devel
```

Comment:
This layer should still be calling `dnf clean all`.  Without it, the package cache will get stored in this layer and removed in the next layer therefore it is image bloat.